### PR TITLE
OKTA-397429  OIE SDK - Scenario 5.1.1 - Sample App E2E Complete - [.NET]

### PR DESCRIPTION
**Pull request by andriizhegurov-okta**

### CODE REVIEW

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/5.1-DirectAuthFacebookPassword.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	And Mary does not have an account in the org but has a Facebook account
+
+  Scenario: 5.1.1: Mary Logs in with Social IDP
+	Given Mary navigates to the Login View
+	When she clicks the Login with Facebook button
+	And logs in to Facebook
+	Then she is redirected to the Root View
+    And she sees a table with her profile info
+    And the cell for the value of email is shown and contains her email
+#    And the cell for the value of name is shown and contains her first name and last name
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/5.1-DirectAuthFacebookPassword.feature:17]: Do we need this line?

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/ITestContext.cs [hunk 2]
```diff
@@ -9,6 +9,7 @@ public interface ITestContext
         Task SetUnenrolledUserAsync(string firstName);
         Task SetActivePasswordAndEmailUserAsync(string firstName);
         Task SetActivePasswordAndSmsUserAsync(string firstName);
+        void SetUnenrolledUserWithFaceBookAccount(string firstName);
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/ITestContext.cs:12]: Nit: Facebook

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/TestContext.cs [hunk 3]
```diff
@@ -17,7 +17,7 @@ public class TestContext : ITestContext, IDisposable
         private readonly IOktaSdkHelper _oktaHelper;
 
         private bool _disposed = false;
-        private const int MaxAttempts = 45; // with delay 1000 ms between attempts, it's 45 seconds of total waiting time 
+        private const int MaxAttempts = 61; // seconds
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Helpers/TestContext.cs:20]: It this keeps failing we might to read this from env var and if it's not there use our default.

File: samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/embedded-auth-with-sdk.csproj [hunk 4]
```diff
@@ -18,7 +18,7 @@
     <MvcBuildViews>false</MvcBuildViews>
     <UseIISExpress>true</UseIISExpress>
     <Use64BitIISExpress />
-    <IISExpressSSLPort>44379</IISExpressSSLPort>
+    <IISExpressSSLPort>8080</IISExpressSSLPort>
```
Comment (laura-rodriguez) [inline samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/embedded-auth-with-sdk.csproj:21]: Let's not change the sample app port.

File: samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/5.1-DirectAuthFacebookPassword.feature (hunk 1 above)
Comment (andriizhegurov-okta) [reply samples/samples-aspnet/embedded-auth-with-sdk/Okta.Idx.Sdk.E2ETests/Features/5.1-DirectAuthFacebookPassword.feature:17]: not really, will remove it 👍

**Review by laura-rodriguez — APPROVED** [review]
A few non-blocker comments but pls revert the port change. Thank you!

**Pull request merged**